doc/release-notes.md

```diff
@@ -134,16 +134,25 @@ The `autocombine` RPC command has been replaced with specific set/get commands (
 
 The minimum supported miniUPnPc API version is set to 10. This keeps compatibility with Ubuntu 16.04 LTS and Debian 8 `libminiupnpc-dev` packages. Please note, on Debian this package is still vulnerable to [CVE-2017-8798](https://security-tracker.debian.org/tracker/CVE-2017-8798) (in jessie only) and [CVE-2017-1000494](https://security-tracker.debian.org/tracker/CVE-2017-1000494) (both in jessie and in stretch).
 
+OpenSSL is no longer used by PIVX Core
 
-#### Build System
 
-OpenSSL is no longer used by PIVX Core
+#### P2P and network changes
+
+- Added NAT-PMP port mapping support via [`libnatpmp`](https://miniupnp.tuxfamily.org/libnatpmp.html)
 
 
 #### Disable PoW mining RPC Commands
 
 A new configure flag has been introduced to allow more granular control over weather or not the PoW mining RPC commands are compiled into the wallet. By default they are not. This behavior can be overridden by passing `--enable-mining-rpc` to the `configure` script.
 
+
+#### New startup options
+
+- The `-natpmp` option has been added to use NAT-PMP to map the listening port. If both UPnP
+  and NAT-PMP are enabled, a successful allocation from UPnP prevails over one from NAT-PMP.
```

Comment:
Could document the new checkbox in the GUI settings as well.

Reply:
yeah, done. also added a note making it explicitly clear that each option requires router support to succeed